src/program/services/scrapers/mediafusion.py

```diff
@@ -134,25 +138,37 @@ def scrape(self, item: MediaItem) -> tuple[Dict[str, str], int]:
         if identifier:
             url += identifier
 
-        response = self.request_handler.execute(HttpMethod.GET, f"{url}.json", timeout=self.timeout)
+        response = self.request_handler.execute(
+            HttpMethod.GET, f"{url}.json", timeout=self.timeout
+        )
         if not response.is_ok or len(response.data.streams) <= 0:
             logger.log("NOT_FOUND", f"No streams found for {item.log_string}")
             return {}
 
         torrents: Dict[str, str] = {}
 
         for stream in response.data.streams:
-            if not hasattr(stream, "description") and hasattr(stream, "title") and "rate-limit exceeded" in stream.title:
-                raise RateLimitExceeded(f"Mediafusion rate-limit exceeded for item: {item.log_string}")
+            if (
+                not hasattr(stream, "description")
+                and hasattr(stream, "title")
+                and "rate-limit exceeded" in stream.title
+            ):
+                raise RateLimitExceeded(
+                    f"Mediafusion rate-limit exceeded for item: {item.log_string}"
+                )
             description_split = stream.description.replace("📂 ", "")
             raw_title = description_split.split("\n")[0]
-            info_hash = re.search(r"info_hash=([A-Za-z0-9]+)", stream.url).group(1)
+            if scrape_type == "series":
+                raw_title = raw_title.split("/")[0]
```

Comment:
hmm.. 

In previous updates I've been pretty back and forth on this one.. Do we want the torrent or do we want the file in these scenarios.. We would prefer the torrent I think because it would have more episodes that might be needed from that same torrent.

Reply:
I'll fix this one. Thanks man!

Reply:
Here it's providing the episode file name. If you preferred to get the episode file name, that's also fine.